app/components/back-button.tsx

```diff
@@ -18,14 +18,9 @@ export const BackButton: FC<Omit<ButtonProps, 'children'>> = ({ onClick, ...prop
   const hasBackState = !!backUrl;
 
   return (
-    <Button
-      variant="unstyled"
-      _hover={{ background: 'none' }}
-      position="absolute"
-      left={0}
-      top="5px"
-      height="34px"
-      width="34px"
+    <IconButton
```

Comment:
`<IconButton />` isn't a button 🤔  This change results in no focus state for the back button.

Reply:
as="button" or as={Button}